workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -120,7 +120,7 @@ private static DownloadResult Download()
                     exitCode = RedirectedProcess.Run(Common.DotNetBinary, ConstructArguments());
                     if (exitCode != 0)
                     {
-                        Debug.LogError($"Failed to download SpatialOS Core Sdk version {Common.CoreSdkVersion}.");
+                        Debug.LogError($"Failed to download SpatialOS Core Sdk version {Common.CoreSdkVersion}. You can use SpatialOS -> Download CoreSDK (force) to retry this.");
```

Comment:
These menu item names should probably be constants interpolated into the string, so this message stays up to date when things invariably change.

Reply:
I agree, although currently trying to be suuuuper low impact with these changes. Ill make a ticket.